doc-site/docs/releasenotes/index.md

```diff
@@ -2,8 +2,29 @@
 title: Release Notes
 ---
 
+# Release Notes
+
 [Full release notes](https://github.com/hyperledger/firefly/releases)
 
+## [v1.3.3 - Mar 24, 2025](https://github.com/hyperledger/firefly/releases/tag/v1.3.3)
+
+What's New:
+- Add new method to provide reliable receipt storage for blockchain plugins
+- Changes in metrics:   
+    - Added new metrics for Data Exchange 
+    - Added a namespace label to existing metrics and change gauges to vectors
+    - Added `monitoring` configuration section and deprecated `metrics` section
```

Comment:
This is a significant one - given this is about endpoint listeners. Probably a separate section to the metrics (as the purpose is actually unrelated to metrics - it's to allow the port that hosts the Open Telemetry endpoint, to also host other things like status endpoints).

Reply:
Yeah, good point

Reply:
So it was already a separate server and we just change the configuration name to make it more readable and makes more sense